sympy/solvers/ode/single.py

```diff
@@ -2615,5 +2617,182 @@ def _get_general_solution(self, *, simplify_flag: bool = True):
         return [sol]
 
 
+class LinearBessel2nd(SingleODESolver):
+    r"""
+    Gives solution of the Bessel differential equation
+
+    .. math :: x^2 \frac{d^2y}{dx^2} + x \frac{dy}{dx} y(x) + (x^2-n^2) y(x)
+
+    if n is integer then the solution is of the form Eq(f(x), C0 besselj(n,x)
+    + C1 bessely(n,x)) as both the solutions are linearly independent else if
+    n is a fraction then the solution is of the form Eq(f(x), C0 besselj(n,x)
+    + C1 besselj(-n,x)) which can also transform into Eq(f(x), C0 besselj(n,x)
+    + C1 bessely(n,x)).
```

Comment:
The equations here should be in LaTeX. If they are supposed to be code rather than equations then they should be in double backticks.